### PR TITLE
Fix creating migration with defaultValue

### PR DESCRIPTION
**Pull request by wsaid**
Fix creating migration issue for column methods used after defaultValues

### CODE REVIEW

**Comment by bizley** [comment]
#18303

File: tests/framework/console/controllers/MigrateControllerTest.php [hunk 1]
```diff
@@ -131,6 +131,15 @@ public function generateMigrationDataProvider()
                     price:money(11,2):notNull,
                     parenthesis_in_comment:string(255):notNull:comment(\'Name of set (RU)\')',
             ],
+            'create_fields_withc_col_method_after_default_value' => [
```
Comment (bizley) [inline tests/framework/console/controllers/MigrateControllerTest.php:134]:
```suggestion
            'create_fields_with_col_method_after_default_value' => [
```

File: tests/framework/console/controllers/MigrateControllerTest.php [hunk 2]
```diff
@@ -131,6 +131,15 @@ public function generateMigrationDataProvider()
...
+                'fields' => 'id:primaryKey,
+                    title:string(10):notNull:unique:defaultValue("test"):after("id"),
+                    body:text:notNull:defaultValue("test"):after("title"),
+                    address:text:notNull:defaultValue("test"):after("body"),
+                    address2:text:notNull:defaultValue(\'te:st\'):after("address"),
+                    address3:text:notNull:defaultValue(\':te:st:\'):after("address2")',
+
```
Comment (bizley) [inline tests/framework/console/controllers/MigrateControllerTest.php:141]: Remove empty line

File: tests/framework/console/controllers/MigrateControllerTest.php [hunk 3]
```diff
@@ -236,6 +245,9 @@ public function generateMigrationDataProvider()
             ['create_title_with_comma_default_values', 'create_test_table', 'test', $params['create_title_with_comma_default_values']],
             ['create_field_with_colon_default_values', 'create_test_table', 'test', $params['create_field_with_colon_default_values']],
 
+            // @see https://github.com/yiisoft/yii2/issues/18303
+            ['create_fields_withc_col_method_after_default_value', 'create_test_table', 'test', $params['create_fields_withc_col_method_after_default_value']],
```
Comment (bizley) [inline tests/framework/console/controllers/MigrateControllerTest.php:249]:
```suggestion
            ['create_fields_with_col_method_after_default_value', 'create_test_table', 'test', $params['create_fields_with_col_method_after_default_value']],
```

**Review by bizley — CHANGES_REQUESTED** [review]

**Review by bizley — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thank you!

**Pull request merged**